package.json

```
@@ -12,7 +12,8 @@
     "bundle": "cd tools/bundler && npm i && npm run bundle",
     "startNewVersion": "newVersion=$npm_config_new_version node scripts/add-new-version.js",
     "lint": "echo 'No linter integrated yet'",
-    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "lint-schemas": "node scripts/validate-schemas-final.js"
```

We discussed about this somewhere but i can't find my comment, so suggesting a rename here directly: 

```suggestion
    "validate-schemas": "node scripts/validate-schemas-final.js"
```

---

> We discussed about this somewhere but i can't find my comment, so suggesting a rename here directly:

Oh, I just remembered this after your comment. Anyways, it is more appropriate naming for this. :100: 

resolved with b48c050